gammapy/datasets/fermi.py

```diff
@@ -215,6 +217,158 @@ def fetch_fermi_diffuse_background_model(filename='gll_iem_v02.fit'):
 
     return filename
 
+class Fermi3FGLObject(object):
+    """
+    Class representing an object in the Fermi 3FGL catalog.
+    """
+
+    from astropy.units import Quantity
```

Comment:
Move the import to the top of the file.
If moving the `EnergyBounds` import causes a circular import problem, OK to leave it here, but the Astropy import can go to the top for sure.